pandas/tests/groupby/test_groupby.py

```diff
@@ -1196,6 +1196,21 @@ def test_groupby_complex():
     tm.assert_series_equal(result, expected)
 
 
+def test_groupby_complex_2():
```

Comment:
A name that describes the issue a little would be great

Note: We add 

# GH#issue number

as the first line of the test